expression/expression.go

```diff
@@ -1254,7 +1254,7 @@ func canScalarFuncPushDown(scalarFunc *ScalarFunction, pc PbConverter, storeType
 func canExprPushDown(expr Expression, pc PbConverter, storeType kv.StoreType, canEnumPush bool) bool {
 	if storeType == kv.TiFlash {
 		switch expr.GetType().Tp {
-		case mysql.TypeEnum, mysql.TypeBit, mysql.TypeSet, mysql.TypeGeometry, mysql.TypeUnspecified:
```

Comment:
Is `tiflash` support `bit` column yet?

Reply:
As we haven't supported push bit into tiflash, we should keep `Typebit` here

Reply:
ok